style.css

```diff
@@ -1,114 +1,215 @@
+.row{
+    display: flex;
+    min-height: 100vh;
+    overflow: hidden;
+}
+
 .left,.right{
-    height: 50vh ;
+    align-items:center;
+    height: 100% ;
+    justify-content:center;
     width: 100%;
 }
 
-@media screen and (min-width:768px){
-    .left,.right{
+.form-2,.l-footer{
```

Comment:
space after comma and space before `{`. The above line can be written as:
`.form-2, .l-footer {`. Follow this to other places as well.